src/core/static-layout.js

```diff
@@ -109,7 +115,7 @@ export function applyStaticLayout(element) {
       toggle(element, false);
       // TODO(jridgewell): Temporary hack while SSR still adds an inline
       // `display: none`
-      element['style']['display'] = '';
+      setStyle(element, 'display', ''); // eslint-disable-line local/no-style-display
```

Comment:
This is fine, but we could probably just remove it entirely. The "temporary" hack was added 3 years ago, and the SSR has been updated.

Reply:
> could _probably_ just remove it entirely

How sure are you about this? I'm happy to remove it, but don't know anything about the context of why it was added.

Reply:
Done. Approve if you are confident/remember context. Otherwise I'll revert